populate_arena.py

```diff
@@ -26,7 +26,7 @@ def populate_arena(devices):
 
         shutil.copytree(src, dst)
 
-        print("\t{0}|{1}".format(prm[0], dst))
+        print("\t{0}|{1}|{2}".format(prm[0], prm[1], dst))
```

Comment:
I'll make that change. I think using tabs will make it easier to visually discern.